plugins/fundchannel.c

```diff
@@ -179,6 +187,7 @@ static struct command_result *tx_prepare_done(struct command *cmd,
 		plugin_err("Unable to parse tx %s", hex);
 
 	/* Find the txout */
+	outnum_found = false;
```

Comment:
Seems like a fixup for a previous commit.  Would be nice to squash it into that one.